backend/src/main/kotlin/org/loculus/backend/config/BackendSpringConfig.kt

```diff
@@ -101,6 +109,25 @@ class BackendSpringConfig {
     @Bean
     fun openApi(backendConfig: BackendConfig) = buildOpenApiSchema(backendConfig)
 
+    @Bean
+    fun s3Config(
+        @Value("\${${BackendSpringProperty.S3_ENABLED}}") enabled: Boolean = false,
+        @Value("\${${BackendSpringProperty.S3_BUCKET_ENDPOINT}}") endpoint: String? = null,
+        @Value("\${${BackendSpringProperty.S3_BUCKET_INTERNAL_ENDPOINT}:#{null}}") internalEndpoint: String? = null,
+        @Value("\${${BackendSpringProperty.S3_BUCKET_REGION}}") region: String? = null,
+        @Value("\${${BackendSpringProperty.S3_BUCKET_BUCKET}}") bucket: String? = null,
+        @Value("\${${BackendSpringProperty.S3_BUCKET_ACCESS_KEY}}") accessKey: String? = null,
+        @Value("\${${BackendSpringProperty.S3_BUCKET_SECRET_KEY}}") secretKey: String? = null,
+    ): S3Config {
+        if (!enabled) {
+            return S3Config(false, null)
+        }
+        if (endpoint != null && region != null && bucket != null && accessKey != null && secretKey != null) {
```

Comment:
Region could be null per S3BucketConfig, see below, so could be removed here, or the nullability could be removed below

Reply:
I've removed the null check here: ab92c6722